internal/health/mock.go

```diff
@@ -0,0 +1,33 @@
+package health
```

Comment:
i'd rename this to `mock_test.go` to exclude it from being built in the binary

Reply:
Unfortunately, if a file is named with `_test.go`, go disallow referencing any of its exported code from another package.